src/Storefront/Resources/views/storefront/component/address/address-form.html.twig

```diff
@@ -9,6 +9,8 @@
 
     {% block component_address_form_company %}
         {% if shopware.config.core.loginRegistration.showAccountTypeSelection %}
+            {% set accountTypeRequired = true %}
+            {# @deprecated tag:6.4.0 - Variable "accounTypeRequired" will be removed without replacement. Use "accountTypeRequired" instead #}
```

Comment:
ok sorry, without replacement is of course wrong :smile: :man_facepalming:  should be :wink: 
```suggestion
            {# @deprecated tag:6.4.0 - Variable "accounTypeRequired" will be removed. Use "accountTypeRequired" instead #}
```

Reply:
I should've reviewed the suggestion from the reviewer more closely :shipit:

Reply:
indeed